### DISCUSSION TITLE
Continue in negative once the timer is finished ?

### DISCUSSION THREAD

**Discussion by GaylordP** [vydimitrov/react-countdown-circle-timer#218] [Q&A] [answered]
Hello and thank you for your fabulous library :)

Is it possible, once the timer is finished (`remainingTime === 0`), that it continues in the negative?
In resume : -1 ; -2 ; -3 ; - 4 ; ...

Thanks in advance :)

**Comment by vydimitrov** [comment, marked as answer]
Hey @GaylordP, I assume that you want the progress bar that shows the remaining time to be gone as well (because when the remaining time is 0 then there is no progress bar). If that is the case, the thing you can do is once it is over, you set the option to repeat the timer and this time set the `colors` prop to be `rgba(0,0,0,0)` so it is transparent and you will need to control the display time to be a negative number, for that you can use the `elapsedTime` prop from the `children` prop and just add `-` or `-Math.floor(elapsedTime)`.

**Comment by GaylordP** [comment]
Thank you very much  ! :)
